lib/mutant/subject/method.rb

```diff
@@ -13,12 +13,12 @@ class Method < self
 
       # Return method name
       #
-      # @return [Symbol]
+      # @return [Expression]
       #
       # @api private
       #
       def name
-        node.children[self.class::NAME_INDEX]
+        node.children.at(self.class::NAME_INDEX)
```

Comment:
Is there a mutation for `#[]` to `#at` that showed you this? I wonder if there are other places in the codebase that could use `#at` instead.